src/cmd/linuxkit/push_packet.go

```diff
@@ -11,6 +10,7 @@ import (
 	"path/filepath"
 	"runtime"
 
+	"github.com/klauspost/compress/gzip"
```

Comment:
Why do we use `klauspots/compress/gzip` here and `klauspost/pgzip` elsewhere?

Reply:
Thanks, it's a mistake